OpenRA.Platforms.Default/ThreadedGraphicsContext.cs

```diff
@@ -557,7 +557,6 @@ class ThreadedTexture : ITextureInternal
 		readonly Func<object> getSize;
 		readonly Action<object> setEmpty;
 		readonly Func<byte[]> getData;
-		readonly Func<object, object> setData1;
```

Comment:
You probably kept it simple to reduce the diff size, but I think renumbering the fields would be nice.